pkgs/tools/misc/bat/default.nix

```diff
@@ -16,7 +16,7 @@ rustPlatform.buildRustPackage rec {
 
   src = fetchFromGitHub {
     owner = "sharkdp";
-    repo = pname;
+    repo = "bat";
```

Comment:
is there a specific reason for this change? `repo = pname` seems to be a very common pattern

Reply:
AFAIK finalattrs hasn't been implemented for buildRustPackage?

Reply:
So there isn't a reason for this change?

Reply:
I believe this is still valid, even without `finalAttrs` pattern. If one needs to overrides `pname`, it will also have an impact on the `src`. This change is preventing that.

Reply:
I won't ask you to revert it but frankly this change is pointless. 

> I don't want to loose too much time on this

I'd ask you to consider that other people may not want `to loose too much time` on stuff like this either.

This PR could have been a one line diff adding mainProgram that could have been merged without discussion as soon as it passed ofborg.

Reply:
I'm dedicating my time for free for this project and every time I edit a PR, I'm trying to use what the "seniors" has been telling me for months when I was submitting my first PRs.

Constantly refining for a greater good the things that I'm "touching", I see that it is not very welcome, point noted, maybe I should avoid dedicating time to this community?